client/signup/config/flows.js

```diff
@@ -109,6 +109,10 @@ function getDestinationFromIntent( dependencies ) {
 		return `/post/${ siteSlug }`;
 	}
 
+	if ( isFSEActive ) {
+		return `/site-editor/${ dependencies.siteSlug }`;
+	}
+
```

Comment:
This will redirect to the Site editor after the site-setup flow... I'm not sure if there are other flows where we'd want to do this.

Reply:
All the site creation flows (e.g. `onboarding`, `free`, etc.) eventually redirect to the `site-setup` flow, so this is the only one we have to worry about 👍